scopes/ui-foundation/ui/start-cmd/cli-output.tsx

```diff
@@ -42,17 +40,17 @@ type state = {
   compiling: boolean;
 };
 
-export type props = {
+export type CliOutputProps = {
```

Comment:
Please add types. `any` is not very useful.

Reply:
I think this is outdated, right?